src/test/java/wbooks/register/register.feature

```diff
@@ -0,0 +1,82 @@
+Feature: enter the required fields to create a new user
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+    * def now = function(){ return java.lang.System.currentTimeMillis() }
+    * def email = 'email' + now() + '@wolox.test.com'
+    * def newUser =
+      """
+      {
+        "user": {
+          "first_name": "testing",
+          "last_name": "register",
+          "email": '#(email)',
+          "password": "12345678",
+          "locale": "en"
+         }
+      }
+      """
+
+  Scenario: register a new user
+
+    Given path 'users'
+      And request newUser
+    When method post
+    Then status 201
+      And match response == { access_token: '#string', renew_id: '#string' }
```

Comment:
It looks better than it is indented

Reply:
You mean removing the `And` indentations? yeah you're right, I check every example and any of those have indentation there, altough I decided to add it anyway haha.
I added it because for me its better to read, separating the request configuration (Given), sending the request (When) and the assertions (Then), but I can change it to no indentation if we are using that as the style standard (but I'd had to do it in every .feature file because right now all of those have this 😛).

Reply:
Changed